locale/en/blog/npm/npm-1-0-link.md

```diff
@@ -107,7 +107,9 @@ I highly doubt that a native Windows node will ever have comparable symbolic lin
 
 When there is a native windows port of Node, if that native windows port has \`fs.symlink\` and \`fs.readlink\` support that is exactly identical to the way that they work on Unix, then this should work fine.
 
-<h2 id="aside_credit_where_credit8217s_due">Aside: Credit where Credit&#8217;s Due</h2>
+But I wouldn't hold my breath. Any bugs about this not working on a native Windows system (ie, not Cygwin) will most likely be closed with `wontfix`.
```

Comment:
Don't think the merge conflict was right here. I'll rebase this to fix it properly